pkg/httputil/httputil.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2017 clair authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package httputil implements common HTTP functionality used throughout the Clair codebase.
+package httputil
+
+import (
+	"net/http"
+
+	"github.com/coreos/clair/pkg/version"
+)
+
+// GetWithUserAgent performs an HTTP GET with the proper Clair User-Agent.
+func GetWithUserAgent(url string) (*http.Response, error) {
+	client := &http.Client{}
+
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	req.Header.Set("User-Agent", "Clair/"+version.Version+" (https://github.com/coreos/clair)")
+
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, err
+	}
+
+	return resp, nil
+}
+
+// Status2xx returns true if the response's status code is success (2xx)
+func Status2xx(resp *http.Response) bool {
+	return resp.StatusCode/100 == 2
+}
```

Comment:
It turns out we have another httputil package. Can you move stuff from there into this?

https://github.com/coreos/clair/blob/master/api/httputil/httputil.go

Reply:
Absolutely. Once this is settled, I'll merge this into master as well. Do you want me to move 'httpuil' in master to pkg?

Reply:
Yup!